.github/workflows/require-allow-edits.yml

```diff
@@ -0,0 +1,14 @@
+name: Require “Allow Edits”
+
+on: [pull_request]
+
+jobs:
+  _:
+    name: "Require “Allow Edits”"
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: ljharb/require-allow-edits@main
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
```
this isn't actually required anymore, i just haven't updated all the client repos yet :-)